### PR TITLE
[Enhancement] Save some tablet properties in StarMgr

### PR DESCRIPTION
**Pull request by sduzh**
Subsequent PRs will leverage the saved properties to optimize import performance.

## Problem Summary:
Fixes # (issue)

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto backported to target branch
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 13 / 17 (76.47%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/lake/LakeTable.java|0|4|00.00%|[167, 168, 169, 173]|
|:large_blue_circle:|com/starrocks/alter/LakeTableAlterJobV2Builder.java|5|5|100.00%|[]|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|4|4|100.00%|[]|
|:large_blue_circle:|com/starrocks/lake/StarOSAgent.java|4|4|100.00%|[]|

**Review by kevincai — APPROVED** [review]

**Review by gengjun-git — APPROVED** [review]

File: fe/fe-core/src/main/java/com/starrocks/lake/LakeTable.java [hunk 1]
```diff
@@ -164,11 +164,13 @@ public Status createTabletsForRestore(int tabletNum, MaterializedIndex index, Gl
                                           long partitionId, long shardGroupId) {
         FilePathInfo fsInfo = getPartitionFilePathInfo();
         FileCacheInfo cacheInfo = getPartitionFileCacheInfo(partitionId);
-
+        Map<String, String> properties = new HashMap<>();
+        properties.put(LakeTablet.PROPERTY_KEY_PARTITION_ID, Long.toString(partitionId));
+        properties.put(LakeTablet.PROPERTY_KEY_INDEX_ID, Long.toString(index.getId()));
```
Comment (wyb) [inline fe/fe-core/src/main/java/com/starrocks/lake/LakeTable.java:169]: put table id

Comment (sduzh) [reply fe/fe-core/src/main/java/com/starrocks/lake/LakeTable.java:169]: Cannot get table id here, leaving this property unset is acceptable for now.

**Review by wyb — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.0

**Pull request merged**